### ISSUE TITLE
Maybe should make the border wider when using the interation mode?

### ISSUE THREAD

**Issue by sd309991** [BiGilSoft/WindowTop#255]
When I use the minimize mode, I try to click the button near the edge of the hided window, but I couldn't do that, as when the mouse just exit the small image, the big window will disappear immediately.

**Comment by gileli121** [comment]
Thanks for your feedback.
You mean that when the interacting mode is enabled and the PiP window is small, it is impossible to interact with the edges of the window (because the app thinks that you want to resize the PiP window), right?

I saw it myself.

**Comment by sd309991** [comment]
yes, I would like to be able to click on the close button within the interation mode.

**Comment by gileli121** [comment]
Make sense.
I will add it to the to-do list of improvements.
